jetstream/statistics.py

```diff
@@ -202,58 +203,80 @@ def flatten_simple_compare_branches_result(
 class BootstrapMean(Statistic):
     num_samples: int = 10000
     confidence_interval: float = 0.95
-    ref_branch_label: str = "control"
+    ref_branch_label: Optional[str] = "control"
 
     def transform(self, df: DataFrame, metric: str) -> StatisticResultCollection:
         critical_point = (1 - self.confidence_interval) / 2
         summary_quantiles = (critical_point, 1 - critical_point)
+        results = []
 
-        ma_result = mozanalysis.bayesian_stats.bayesian_bootstrap.compare_branches(
-            df,
-            col_label=metric,
-            ref_branch_label=self.ref_branch_label,
-            num_samples=self.num_samples,
-            individual_summary_quantiles=summary_quantiles,
-        )
+        if self.ref_branch_label is None:
+            ref_branch_list = df.branch.unique()
+        else:
+            ref_branch_list = [self.ref_branch_label]
+
+        for ref_branch in ref_branch_list:
```

Comment:
This feels a bit repetitive since it's _almost_ the same for every statistic. Maybe at some point in the future we can find some cool abstraction for this.